### PR TITLE
Update README example

### PR DESCRIPTION
**Pull request by ricardoV94**
Some docstrings and README changes that were missing from #57

### CODE REVIEW

File: aeppl/joint_logprob.py [hunk 1]
```diff
@@ -26,9 +26,8 @@ def factorized_joint_logprob(
 ) -> Dict[TensorVariable, TensorVariable]:
     r"""Create a graph representing the joint log-probability/measure of a graph.
```
Comment (brandonwillard) [inline aeppl/joint_logprob.py:27]:
```suggestion
    r"""Create a map between variables and their log-probabilities such that the sum is their joint log-probability.
```

File: aeppl/joint_logprob.py [hunk 2]
```diff
@@ -47,11 +46,11 @@ def factorized_joint_logprob(
         Y \sim& \operatorname{N}(0, \sigma^2)
 
     If we create a value variable for ``Y_rv``, i.e. ``y = at.scalar("y")``,
```
Comment (brandonwillard) [inline aeppl/joint_logprob.py:48]:
```suggestion
    If we create a value variable for ``Y_rv``, i.e. ``y_vv = at.scalar("y")``,
```

This only adds the `*_vv` suffix convention for value variables.

File: aeppl/joint_logprob.py [hunk 3]
```diff
@@ -47,11 +46,11 @@ def factorized_joint_logprob(
...
-    the graph of ``joint_logprob({Y_rv: y})`` is equivalent to the
-    conditional probability :math:`\log p(Y = y \mid \sigma^2)`.  If we specify
-    a value variable for ``sigma2_rv``, i.e. ``s = at.scalar("s2")``, then
-    ``joint_logprob({Y_rv: y, sigma2_rv: s})`` yields the joint
-    log-probability
+    the graph of ``factorized_joint_logprob({Y_rv: y})`` is equivalent to the
+    conditional probability :math:`\log p(Y = y \mid \sigma^2)`, with a stochastic
+    ``sigma2_rv``. If we specify a value variable for ``sigma2_rv``, i.e.
+    ``s = at.scalar("s2")``, then ``factorized_joint_logprob({Y_rv: y, sigma2_rv: s})``
+    yields the joint log-probability of the two variables.
```
Comment (brandonwillard) [inline aeppl/joint_logprob.py:53]:
```suggestion
    the graph of ``factorized_joint_logprob({Y_rv: y_vv})`` is equivalent to the
    conditional probability :math:`\log p(Y = y \mid \sigma^2)`, with a stochastic
    ``sigma2_rv``. If we specify a value variable for ``sigma2_rv``, i.e.
    ``s_vv = at.scalar("s2")``, then ``factorized_joint_logprob({Y_rv: y_vv, sigma2_rv: s_vv})``
    yields the joint log-probability of the two variables.
```

This only adds the `*_vv` suffix convention for value variables.

**Review by brandonwillard — CHANGES_REQUESTED** [review]
Since `joint_logprob` will be a likely point-of-entry for most people, we should make its signature more explicit and find a way to reuse parts of the docstring from `factorized_joint_logprob`.

Is anyone aware of a good approach to the latter that doesn't involve copy-pasting text?

**Comment by ricardoV94** [comment]
> Since `joint_logprob` will be a likely point-of-entry for most people, we should make its signature more explicit and find a way to reuse parts of the docstring from `factorized_joint_logprob`.
> 
> Is anyone aware of a good approach to the latter that doesn't involve copy-pasting text?

Only thing I could find was the `functools.wraps` but doesn't really apply to this. So unless someone has any ideas, either copy-paste or place the docstrings in the `joint_logprob`

**Review by brandonwillard — APPROVED** [review]

**Pull request merged**